### PR TITLE
Aws ec2 close port 11211

### PR DESCRIPTION
**Pull request by sreedevikr**
This commit includes the changes for the custom remediation job to restrict public access to Memcache UDP port (11211) of EC2 instance. Could you please also share the details about the rule Id. We have not updated the ruleid currently.

### CODE REVIEW

**Review by Manmohan-Tyagi — DISMISSED** [review]

**Review by kshrutik — CHANGES_REQUESTED** [review]

**Comment by kshrutik** [comment]
Please add unit-test

File: remediation_worker/jobs/aws_ec2_close_port_11211/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Close Port 11211 for all Security Groups associated with an EC2 Instance
+
+This job blocks public access to port 11211 for both IPv4 and IPv6 for all security groups associated with an EC2 instance by removing all the ingress security group rules containing port 11211 in the port range and source as "0.0.0.0/0" or "::/0".
+
+### Applicable Rule
+
+##### Rule ID:
+5c8c263e7a550e1fb6560c3b
```
[the remark this answers is not in the document]
Comment (sreedevikr) [reply remediation_worker/jobs/aws_ec2_close_port_11211/README.md]: Thanks @kshrutik for the review comments, I have updated the ruleid.

**Comment by sreedevikr** [comment]
> Please add unit-test

Thanks @kshrutik for review comments, I have included the unit test case.

**Comment by kshrutik** [comment]
Also, can you please update the main readme file and the tox file? Run the tox file and verify if the unit tests are passing.
Refer this [PR](https://github.com/vmware-samples/secure-state-remediation-jobs/pull/97)

**Comment by sreedevikr** [comment]
@kshrutik Thanks for the review comments , I have updated the readme and the tox file.

File: README.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ The table below lists all the supported jobs with their links.
 |   39.  	| 5c8c25f37a550e1fb6560bca              | EC2 VPC default security group should restrict all access                               | [aws-ec2-default-security-group-traffic](remediation_worker/jobs/aws_ec2_default_security_group_traffic) 	|
 |   40.  	| 5c8c260b7a550e1fb6560bf4              | IAM password policy should set a minimum length                               | [aws-iam-password-policy-min-length](remediation_worker/jobs/aws_iam_password_policy_min_length) 	|
 |   41.  	| 5c8c26107a550e1fb6560bfc              | IAM password policy should prevent password reuse                               | [aws-iam-password-reuse-prevention](remediation_worker/jobs/aws_iam_password_reuse_prevention) 	|
-|   42.  	| 7fe4eb28-3b82-11eb-adc1-0242ac120002              | IAM server certificates that are expired should be removed                               | [aws-iam-server-certificate-expired](remediation_worker/jobs/aws_iam_server_certificate_expired) 	|
```
Comment (kshrutik) [inline README.md:155]: You have replaced the entry for - "IAM server certificates that are expired should be removed". Please add it again.

**Review by kshrutik — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sreedevikr) [reply README.md:155]: Sorry about that, Thanks @kshrutik for pointing that out. I have corrected this. Thanks

**Review by kshrutik — APPROVED** [review]

**Review by kshrutik — APPROVED** [review]

**Pull request merged**